molsim/classes.py

```diff
@@ -1130,6 +1130,8 @@ def __init__(
                     use_obs = False, # flag for line profile simulation to be done with observations
                     add_noise = False, #flag for whether to add noise 
                     noise = None, # Noise level in native units of the simulation to add
+                    tau_threshold = 1000., #Set upper threshold at which to ignore lines for fitting (Samer)
+                    eup_threshold = 0. #Set lower eup threshold [K] at which to exclude transitions (Samer)
```

Comment:
Does this imply that by default, everything with an upper state energy >= 0. will have `tau` set to zero?

Reply:
Everything with an upper state energy greater than 0 will be included. It's meant to be a lower limit

Reply:
Basically I don't want the default behavior of this function to change - can you guarantee someone else using this function won't have their simulations completely messed up by the default value?

Reply:
Why not set the defaults for both to None, and add logic to your added code to just skip any changes if these values are None?  That would ensure no weird edge cases you haven't thought of.   I can think of several cases where 1e6 would not be good (not because that's a reasonable value, but because someone might want to test making some lines super optically thick, only to see them disappear).

Reply:
Done!